### PR TITLE
Add Prelude 0.6

### PR DESCRIPTION
**Pull request by t-imamichi**
### Summary

- Added prelude of v0.6.
- Added migration guide to v0.6.

### CODE REVIEW

File: docs/migration/01_migration_guide_to_v0.5.rst [hunk 1]
```diff
@@ -55,23 +55,14 @@ an extension that allows users to access the eigen states.
 
 The following is the corresponding table.
 
-+-------------------------------------+--------------------------------+
-| Former algorithm                    | New algorithm                  |
-+=====================================+================================+
-| ``qis                               | ``qiskit.                      |
```
Comment (t-imamichi) [inline docs/migration/01_migration_guide_to_v0.5.rst:61]: I rewrote the table with cvs-table because the wrong line breaks generates redundant spaces.

**Comment by woodsp-ibm** [comment]
Can update this release note: switch-to-qiskit-algorithms-b0af38fe547bf5b5.yaml. It has a bit which is also in the prelude but can we change `Qiskit Terra` to simply `Qiskit` 

> Qiskit Optimization 0.6 switches from qiskit.algorithms of Qiskit Terra to [Qiskit Algorithms](https://qiskit.org/ecosystem/algorithms/). Qiskit Optimization 0.6 drops supports of the former algorithms based on qiskit.algorithms, QuantumInstance, and Opflow of Qiskit Terra.

This is also explained in the blog but I guess its ok to have this short text there too which was done earlier.

**Comment by t-imamichi** [comment]
Thanks. I updated it.

File: releasenotes/notes/switch-to-qiskit-algorithms-b0af38fe547bf5b5.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 prelude: >
-    Qiskit Optimization 0.6 switches from ``qiskit.algorithms`` of Qiskit Terra
+    Qiskit Optimization 0.6 switches from ``qiskit.algorithms`` of Qiskit
     to `Qiskit Algorithms <https://qiskit.org/ecosystem/algorithms/>`_.
     Qiskit Optimization 0.6 drops supports of the former algorithms based on
     ``qiskit.algorithms``, ``QuantumInstance``, and Opflow of Qiskit Terra.
```
Comment (woodsp-ibm) [inline releasenotes/notes/switch-to-qiskit-algorithms-b0af38fe547bf5b5.yaml:6]:
```suggestion
    ``qiskit.algorithms``, ``QuantumInstance``, and Opflow of Qiskit.
```

**Pull request merged**